### PR TITLE
DOCSP-30335: Add Manage Multi-User Applications page to Kotlin

### PR DESCRIPTION
**Pull request by cbullinger**
## Pull Request Info

Add new Manage Multi-User Applications page based on [existing Java](https://www.mongodb.com/docs/realm/sdk/java/users/multi-user-applications/) page

### Jira

- https://jira.mongodb.org/browse/DOCSP-30335

### Staged Changes

- [Manage Multi-User Applications - Kotlin SDK](https://preview-mongodbcbullinger.gatsbyjs.io/realm/docsp-30335-multi-user-app/sdk/kotlin/users/multi-user-applications/)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/kotlin/users/multi-user-applications.txt [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This page describes how to manage multiple Atlas App Services users 
+within a single device using the Kotlin SDK. To learn how to authenticate and 
+log in users to an App, refer to :ref:`<kotlin-authenticate-users>`.
```
Comment (lindseymoore) [inline source/sdk/kotlin/users/multi-user-applications.txt:15]:
```suggestion
log in users to an app, refer to :ref:`<kotlin-authenticate-users>`.
```
Unless you include the full title before it (Atlas App Services App), capitalizing app might be confusing to readers without the extra context. Also, seems a bit inconsistent when capital 'App' or lowercase 'app' or 'application' is used, so I'd just go with lower case.

File: source/sdk/kotlin/users/multi-user-applications.txt [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+**active user** even if multiple users are logged in simultaneously.
+There can only be one active user at a time, and the active user is 
+is associated with all outgoing requests. 
+
+User Accounts 
+-------------
+
+A **user account** represents a single, distinct user of your application. 
+App Services creates an associated unique ``User`` object when a user first 
+successfully authenticates and logs in to an App on a device. After log in,
```
Comment (lindseymoore) [inline source/sdk/kotlin/users/multi-user-applications.txt:28]:
```suggestion
successfully authenticates and logs in to an app on a device. After log in, 
```

File: source/sdk/kotlin/users/multi-user-applications.txt [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+User Accounts 
+-------------
+
+A **user account** represents a single, distinct user of your application. 
+App Services creates an associated unique ``User`` object when a user first 
+successfully authenticates and logs in to an App on a device. After log in, 
+the SDK saves the user's information and keeps track of the 
+user's state on the device. This data remains on the 
+device, even if the user log out, unless you actively remove the user
```
Comment (lindseymoore) [inline source/sdk/kotlin/users/multi-user-applications.txt:31]:
```suggestion
device, even if the user logs out, unless you actively remove the user 
```
typo

File: source/sdk/kotlin/users/multi-user-applications.txt [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+               
+      User on Device 651330cebe1d42b24b8d510f: 65133e1357aaf22529343c1b
+      User on Device 651330cebe1d42b24b8d510f: 65133e130075a51f12a9e635
+
+Log a User Out
+--------------
+
+You can log a logged-in user out of an app using the 
+`user.logOut() <{+kotlin-sync-prefix+}io.realm.kotlin.mongodb/-user/log-out.html>`__ 
+method. Once logged out, the user is still stored on the device, but
```
Comment (lindseymoore) [inline source/sdk/kotlin/users/multi-user-applications.txt:134]:
```suggestion
method. Once logged out, the user is still stored on the device but
```

File: source/sdk/kotlin/users/multi-user-applications.txt [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+must log back in to use the app. 
+
+In the following example, Joe is currently logged-in as the current user. 
+After we log Joe out of the app, we confirm that he is still stored on 
+the device as a user and Emma is now the current user:
```
Comment (lindseymoore) [inline source/sdk/kotlin/users/multi-user-applications.txt:139]:
```suggestion
the device as a user and that Emma is now the current user: 
```
suggestion

**Review by lindseymoore — APPROVED** [review]
Small comments, otherwise LGTM!

File: source/sdk/kotlin/users/multi-user-applications.txt (hunk 1 above)
Comment (cbullinger) [reply source/sdk/kotlin/users/multi-user-applications.txt:15]: I agree and don't love the standard, but I've been told it's capital A "App" when it's connecting to an Atlas App Services App (vs. a client "app" using local realm). I'll go through this page and make sure it's consistent.

Comment (cbullinger) [reply source/sdk/kotlin/users/multi-user-applications.txt:15]: oof, yeah, it's really inconsistent on this page. Good catch

**Pull request merged**